ImperatorToCk3/Source/ImperatorToCK3Converter.h

```diff
@@ -0,0 +1,6 @@
+#ifndef IMPERATORTOCK3_CONVERTER_H
```

Comment:
There's a difference in casing between this filename and the cpp file. (CK3 vs Ck3). Make that consistent.

Reply:
I want to use CK3 everywhere (not in this PR though), so I made it "ImperatorToCK3Converter".